docs/reference/workloads.md

```diff
@@ -198,15 +208,9 @@ functions:
     image: docker.io/stefanprodan/kubesec:v2.1
     skip_build: true
     annotations:
-      com.openfaas.health.http.path: "/healthz"
-      com.openfaas.health.http.initialDelay: "30s"
-      com.openfaas.health.http.periodSeconds: 5s
+      com.openfaas.ready.http.path: "/_/ready"
+      com.openfaas.ready.http.initialDelay: "30s"
+      com.openfaas.ready.http.periodSeconds: 5s
 ``` 
 
 > Note: The initial delay value must be a valid Go duration e.g. `80s` or `3m`.
-
-Readiness probes use the same HTTP path as the health check by default. A custom ready path can be set with the following annotation:
-
-* `com.openfaas.ready.http.path`
-
-The `timeoutSeconds` value for liveness and readiness probes can be set globally for the installation: [OpenFaaS chart reference](https://github.com/openfaas/faas-netes/tree/master/chart/openfaas#faas-netes--operator). A global `initialDelaySeconds` and `periodSeconds` can also be set for the installation, any overrides set for these two values in annotations will take precedence.
```

Comment:
Where did this go?

Reply:
This has been moved to the start if this section and reworded in a more general way.

Reply:
I can't see `timeoutSeconds` mentioned anymore in this diff?

Reply:
I made another change to this section to explicitly mention the values that can be configured.

Reply:
Thanks, a customer needed this, so we should not remove it from the docs as it would probably be confusing for them.